suite-native/navigation/src/components/ScreenHeader.tsx

```diff
@@ -2,33 +2,47 @@ import React from 'react';
 import { useNavigation } from '@react-navigation/native';
 
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { Box } from '@suite-native/atoms';
+import { Box, Text } from '@suite-native/atoms';
 import { TouchableOpacity } from 'react-native';
 import { Icon } from '@trezor/icons';
 
+type ScreenHeaderProps = {
+    title?: string;
+};
+
+const goBackButtonSize = 40;
```

Comment:
```suggestion
const GO_BACK_BUTTON_SIZE = 40;
```
This is a primitive constant used as default. I think we use SCREAMING_SNAKE_CASE for naming them

Reply:
[6f18eaf](https://github.com/trezor/trezor-suite/pull/5693/commits/6f18eaf7956614d05a8f03a0f397ee2a2d8ced05)